index.js

```diff
@@ -1 +1,28 @@
 // Code your solution in this file!
+
+const drivers = ['Antonia', 'Nuru', 'Amari', 'Mo'];
+
+const returnFirstTwoDrivers = function(drivers) {
+    return drivers.slice(0,2);
+};
+
+const returnLastTwoDrivers = function(drivers) {
+    return drivers.slice(2);
```

Comment:
This will grab everything after the 2nd element of the array. `.slice(-2)` will grab the last two elements of the array. In an array with 4 objects, both will return the same thing, the difference would be present with any other number of elements though.